src/utilities/__tests__/extendSchema-test.js

```diff
@@ -359,6 +359,33 @@ describe('extendSchema', () => {
     expect(printExtensionNodes(someScalar)).to.deep.equal(extensionSDL);
   });
 
+  it('extends scalars by adding specifiedBy directive', () => {
+    const schema = buildSchema(`
+      type Query {
+        foo: Foo
+      }
+
+      scalar Foo
+
+      directive @foo on SCALAR
+    `);
+    const extensionSDL = dedent`
+      extend scalar Foo @foo
+
+      extend scalar Foo @specifiedBy(url: "https://example.com/foo_spec")
+    `;
+
+    const extendedSchema = extendSchema(schema, parse(extensionSDL));
+    const foo = assertScalarType(extendedSchema.getType('Foo'));
+
+    expect(foo.toConfig().specifiedByUrl).to.equal(
```

Comment:
You don't need `toConfig` here, just do:
```suggestion
    expect(foo).to.include({
```

Reply:
Fixed in b44e2c9e